### PR TITLE
amenity=bicycle_repair_station symbol (public 24/7 tools for fixing bikes)

### PR DESCRIPTION
**Pull request by brycenesbitt**
An amenity=bicycle_repair_station is usually available 24/7,and hosts tools for repairs of bicycles.  There are now over 500 mapped.  Many people are not aware these stations even exist, thus the mapping.

The svg file is licence compatible.

### CODE REVIEW

**Comment by mkoniecz** [comment]
Please, add before/after images.

**Comment by pnorman** [comment]
Does Mapnik even support all the stuff you're using in that SVG?

File: symbols/bicycle_repair_tools.svg [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+     id="g3138"
+     transform="matrix(0.98757845,0,0,1,-171.61601,7.0110373)"
+     inkscape:export-xdpi="4.2362185"
+     inkscape:export-ydpi="4.2362185">
+    <circle
+       transform="matrix(0.612193,0,0,0.612193,156.40859,178.29391)"
+       r="161"
+       cy="409"
+       cx="215"
+       style="fill:none;stroke:#000000;stroke-width:40px;stroke-opacity:1"
```
Comment (pnorman) [inline symbols/bicycle_repair_tools.svg]: I've had problems with stroke before in Mapnik

**Comment by pnorman** [comment]
Regardless of the technical issues with the SVG, I'm against adding `amenity=bicycle_repair_station`. I feel it's too specialized and is similar to some other bicycle-specific features we've decided not to render (#591).

I'd like to see the example images covering the proposed rendering in a variety of settings.

**Comment by mkoniecz** [comment]
> is similar to some other bicycle-specific features we've decided not to render (#591).

In case of #591 I think that current situation is that there is no consensus for small ones, but gigantic ones should be somehow rendered.

In this case I am unsure what counting my positive bias toward anything cycling-related may be counted as being against rendering amenity=bicycle_repair_station.

**Comment by math1985** [comment]
I see no harm in rendering this, but only from high zoom levels to avoid clutter (z18 or z19). I think z17 is too early for a general purpose map.

**Comment by nebulon42** [comment]
@brycenesbitt re consistency: Here is my suggestion for bike repair that is consistent with bike rental (which is already in master) repair-bicycle-18 [nebulon42/osmic/repair-bicycle-18.svg](https://github.com/nebulon42/osmic/blob/master/transport/repair-bicycle-18.svg)

Here is a version without halo (preferred by maintainers): https://gist.githubusercontent.com/nebulon42/bcae5a3d535ff862a99a/raw/6dda54bc284e36fb430dd6e130ed6a2b668da92d/repair-bicycle-16.svg

**Comment by math1985** [comment]
I think 500 items is still too little for it too deserve its own icon. Maybe in the future! Thanks anyway @brycenesbitt!

**Comment by brycenesbitt** [comment]
This is a worldwide feature, growing, with the capability of drawing new mappers to the project.  A fraction of the existing stations are mapped, but the feature will still be far more rare than toilets or bike parking:
selection_189

**Comment by kocio-pl** [comment]
Now the number of uses is much higher (2800), the new icon has been proposed in a duplicate ticket, but we already have it in Osmic: 

https://github.com/gmgeo/osmic/blob/master/shop/repair-bicycle-14.svg

**Comment by kocio-pl** [comment]
Oh, sorry - it's the old PR, not an issue...

**Pull request closed without merging**